app/perfaddresscache.h

```diff
@@ -42,11 +42,13 @@ class PerfAddressCache
 
     struct SymbolCacheEntry
     {
-        SymbolCacheEntry(quint64 offset = 0, quint64 value = 0, quint64 size = 0, const QByteArray &symname = {})
+        SymbolCacheEntry(quint64 offset = 0, quint64 value = 0, quint64 size = 0, const QByteArray &symname = {}, bool relocated = false, quint64 adjust = 0)
```

Comment:
Since this is in a central cache which is shared across processes even, are you sure that the adjust value is always the same? Couldn't it be influenced by ASLR or such? Can you verify that this value really is going to be the same across all processes that link to a given library?

Reply:
Do you have any ideas how could I verify this? Probably it is also the matter of future development?

Reply:
run `perf record -a --call-graph dwarf -- sleep 5` and then look at the symbols for e.g. libc which should be encountered in many processes - do they all report the same adjust values?

and no, this fundamental - this must not slip in and left for the future to be fixed, it has to be done right

Reply:
I have got the same adjust values.

Reply:
perfect, thanks for checking